editor/editor_help.cpp

```diff
@@ -2193,6 +2205,12 @@ void EditorHelp::_update_doc() {
 				}
 				has_prev_text = true;
 				_add_text(descr);
+				// Add copy note to built-in properties returning Packed*Array.
+				if (!cd.is_script_doc && packed_array_types.has(prop.type)) {
+					class_desc->add_newline();
+					class_desc->add_newline();
+					_add_text("[b]" + TTR("Note:") + "[/b] " + vformat(TTR("The returned array is [i]copied[/i] and any changes to it will not update the original property value. See [%s] for more details."), prop.type));
```

Comment:
```suggestion
					_add_text(vformat(TTR("[b]Note:[/b] The returned array is [i]copied[/i] and any changes to it will not update the original property value. See [%s] for more details."), prop.type));
```
Is there a reason to translate `Note:` separately? IME, concatenating translated strings should be avoided due to different grammars etc.

Reply:
Copied from elsewhere, forget where